### ISSUE TITLE
Extensions should not be blocked

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#19]
```
What steps will reproduce the problem?
1. Install extension "Firebug"
2. After install press F12 on blocked site

What is the expected output? What do you see instead?
Expected: Firebug window at the bottom
Instead: Chrome web developer toolbar

What version of the product are you using? On what operating system?
ScriptNo 1.0.546 on Chrome 14, OS: Windows 7

Please provide any additional information below.
Extensions should not be blocked by ScriptNo or sth. like 
"chrome-extensions://*" should be added to the whitelist, but is an invalid 
domain. ScriptNo detects domains like "about" and "<ID of firebug plugin>", but 
they cannot be added to the whitelist.

```

Original issue reported on code.google.com by `norb...@gmail.com` on 5 Oct 2011 at 7:51
- Merged into: #13

**Comment by GoogleCodeExporter** [comment]
```
I agree with this. At least make it an option (should be allowed by default.)

In the event that an extension is compromised it can be useful but otherwise 
it's no good.
```

Original comment by `ColinTre...@gmail.com` on 19 Oct 2011 at 8:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 14 Nov 2011 at 3:56

**Comment by GoogleCodeExporter** [comment]
```
This will be addressed in the next release! Sorry for the delay, I've been out 
of the country for a month and a half.
```

Original comment by `andr...@gmail.com` on 14 Nov 2011 at 3:58
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 14 Nov 2011 at 4:01
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Partially addressed in v1.0.5.47; however, the issue still exists due to (what 
I believe to be) the inline script mitigation function. I'll continue looking 
into this.
```

Original comment by `andr...@gmail.com` on 14 Nov 2011 at 8:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 17 Nov 2011 at 2:43
- Changed state: **Duplicate**
